src/bookmarks/background/addition.js

```diff
@@ -50,7 +50,14 @@ export async function createNewPageForBookmark(id, bookmarkInfo) {
     }
 }
 
-export async function createBookmarkByExtension(url) {
+/**
+ * TODO: Decided if we actually need these bookmark docs in Pouch; I don't think they're being used for anything.
+ *
+ * @param {string} url URL to generate page ID from to associate bookmark with.
+ * @throws {Error} Error thrown if page for supplied URL not indexed.
+ * @returns {Promise<void>}
+ */
+export async function createBookmarkByUrl(url) {
```

Comment:
As we are using this function for the bookmark from popup as well so  I think there is no case for when there doesn't exist pageDoc.